<ui version="4.0" >
 <class>ViewDockWidgetUi</class>
 <widget class="QDockWidget" name="ViewDockWidgetUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Views</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents" >
   <layout class="QGridLayout" name="gridLayout" >
    <property name="margin" >
     <number>3</number>
    </property>
    <item row="5" column="0" colspan="2" >
     <widget class="QGroupBox" name="mAdditionalViewGroupBox" >
      <property name="title" >
       <string>Additional Views</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="1" >
        <widget class="QPushButton" name="mNewViewButton" >
         <property name="text" >
          <string>New</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="mEditViewButton" >
         <property name="text" >
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item rowspan="4" row="0" column="0" >
        <widget class="QListWidget" name="mViewWindowListWidget" />
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="mDeleteViewButton" >
         <property name="text" >
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" >
     <widget class="QWidget" native="1" name="mFOVPane" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mViewWindowListWidget</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>mEditViewButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>198</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>417</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
